src/common/featuremanager/FeatureManagerService.js

```diff
@@ -1029,6 +1046,25 @@
         //Get the feature info for each layer that supports it
         goog.array.forEach(layers, function(layer, index) {
           var source = layer.getSource();
+          if (layer.get('metadata').searchLayer || layer.get('metadata').searchResults) {
+            var layerInfo = {};
+            layerInfo.features = [];
+            // 8 is the radius of the circle used for search result points
+            var resolution = view.getResolution() * 8;
+            goog.array.forEach(source.getFeatures(), function(feature) {
+              if (service_.coordinatesWithinRange(feature.getGeometry().getCoordinates(), evt.coordinate, resolution)) {
+                layerInfo.features.push(feature);
+              }
+            });
```

Comment:
Using source.getFeatureAtCoordinate() was not working because I want to grab the area slightly around the click event, hence this workaround. Is this the best way left to do this? I fear this is inefficient due to naively searching through every feature in the search layers for matches.